projects/hitl/software/hitl/ft4222_proxy.py

```diff
@@ -0,0 +1,199 @@
+"""
+Proxy to access FT4222 Hardware. In normal use, these functions shouldn't be
+accessed directly, but instead through the IOController
+"""
+import logging
+import os
+
+from utils import map_to_human, map_to_machine
+
+try:
+    import ft4222
+except ModuleNotFoundError:
+    # We haven't setup the ft4222 library yet...
+    # See https://awenstrup.github.io/setup.html
+    ft4222 = None
```

Comment:
Can anything be done if the FT4222 library isn't set-up? If not, we might want a more fatal error that terminates execution.

Reply:
So this was in the old code, the rationale here is that you can still run a HitL object even if you don't have both the IO device and the CAN device connected if your tests only need one or you're just testing a subset of functionality or something like that.

So it gets initialized to None and then in all of the functions there's a quick check to see if it's None and if it is than it just logs an error and returns but doesn't crash the program. There's a similar thing in the CAN Controller as well. Open to changing it though, that's not something that's set in stone